### ISSUE TITLE
Spelling Typo

### ISSUE THREAD

**Issue by delia925** [featdd/dpn_glossary#202]
Hi I hate to be the one, but theres a typo (Zusfällig):
````
<trans-unit id="tx_dpnglossary.wizard_preview_random_title">
        <target>Glossar Vorschau (Zusfällig)</target>
</trans-unit>````

**Comment by bthwrlds** [comment]
using version 5.0

**Comment by featdd** [comment]
Hi @delia925,

thanks for the hint, fixed that here: c09abef4b270fb659eb6e860782bdbbf5696592c

Greetings
Daniel
